### DISCUSSION TITLE
Switching screens

### DISCUSSION THREAD

**Discussion by Darth-218** [input-leap/input-leap#1905] [Q&A] [answered]
I use dwm as a window manager on both my machines and since dwm allows no need for mouse usage, is there a shortcut to switch between machines instead of moving the mouse to move around?

**Comment by herboh** [comment, marked as answer]
Yes, it's possible to set hotkeys in the server config on the GUI to have the action of switching to a screen. It may be possible to set these binds from a config file with `keystroke ` but I don't have experience.
